src/BaselineOfQA/BaselineOfQA.class.st

```diff
@@ -13,7 +13,11 @@ BaselineOfQA >> baseline: spec [
 	repository := self packageRepositoryURLForSpec: spec.
 
 	spec for: #common do: [
-		spec package: 'NautilusRefactoring'.
+		
+		spec package: 'Refactoring-UI'.
+		spec
+			package: 'NautilusRefactoring'
+			with: [ spec requires: #( 'Refactoring-UI' ) ].
```

Comment:
What is the remaining dependency of `NautilusRefactoring` left in here? Ideally we want to get rid of it and only use new `Refactoring-*` packages. It would be nice to open an issue with rest of required migrations.